CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Scheduling failure events are now emitted on unsuccessful scheduling attempts.
 - Default Javascript function signatures to `encodeDownlink()`, `decodeUplink()` and `decodeDownlink()`.
 - Default Class B timeout is increased from 1 minute to 10 minutes as was originally intended.
+- Update Go to 1.15
```

Comment:
IMO this doesn't belong in the changelog

Reply:
There can be (performance) regressions or improvements caused by updating the runtime. For example, the deprecated X.509 CN handling.

Also we use this project as module.  For external dependency management this is also useful information.